packages/ai-sdk/src/plugin.ts

```diff
@@ -0,0 +1,24 @@
+import type { ProviderV2 } from '@ai-sdk/provider';
+import type { experimental_MCPClient as MCPClient } from '@ai-sdk/mcp';
+import { SimplePlugin } from '@temporalio/plugin';
+import { createActivities } from './activities';
+
+export interface AiSDKPluginOptions {
+  modelProvider: ProviderV2;
+  mcpClientFactory?: (args?: any) => Promise<MCPClient>;
+}
+
+/**
+ * A Temporal plugin that integrates AI SDK providers for use in workflows.
+ * This plugin creates activities that allow workflows to invoke AI models.
+ *
+ * @experimental The AI SDK plugin is an experimental feature; APIs may change without notice.
+ */
+export class AiSDKPlugin extends SimplePlugin {
```

Comment:
nil: This is inconsistent. AI and SDK are both acronyms, and so both shoujld either be written with only the first letter in capital, or both entirely in uppercase.

Reply:
Same for options.